spec/lib/guardian_spec.rb

```diff
@@ -98,6 +98,38 @@
     fab!(:user) { Fabricate(:user) }
     fab!(:post) { Fabricate(:post) }
 
+    describe "an anonymous user" do
+      before { SiteSetting.allow_anonymous_posting = true }
```

Comment:
Hmm, I've never seen this convention before.  Wondering if you want to mock this instead?
```suggestion
      before do
        allow(SiteSetting).
          to receive(:allow_anonymous_posting).
          and_return(true)
      end
```